.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm cherry-picking this over from https://github.com/openedx/openedx-learning/pull/293